src/ui_framework/README.md

```diff
@@ -0,0 +1,57 @@
+# Kibana UI Framework
+
+## Development
+
+* Start development server `npm run uiFramework:start`.
+* View docs on `http://localhost:8080/`.
```

Comment:
8080 is a common, and the default port for webpack dev server. We will probably want to change this in the future to avoid collisions.